server/core/user.go

```diff
@@ -8,7 +8,7 @@ type (
 		ID       uint    `gorm:"primary_key;auto_increment;not null" json:"id"`
 		Email    string  `gorm:"not null;size:255;unique_index" json:"email"`
 		Password string  `gorm:"not null;size:255;column:password" json:"-"`
-		Name     string  `gorm:"not null;size:255" json:"name"`
+		Name     string  `gorm:"not null;size:255;unique_index" json:"name"`
```

Comment:
this is not meant to be username originally, this is full name such as `John Smith`, I don't mind if you add additional property for `username` though.

Reply:
OK, I'm thinking to add "Login" field so that there won't be any confusion with "Name" later (and we wouldn't need to rename all occurrences of the Name -> Full Name later)